addons/event_hr_skills/models/event_registration.py

```diff
@@ -0,0 +1,42 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+
+class EventRegistration(models.Model):
+    _inherit = 'event.registration'
+
+    def _create_resume_lines(self):
+        resume_line_vals_list = []
+        for registration in self:
```

Comment:
Try to simplify loops with easy reject criterions, like
```suggestion
        for registration in self.filtered(lambda r: r.state == 'done' and r.partner_id):
```

Reply:
To be honest here because the rejection is a bit long/complicated I prefer to unroll it in an `if ...: continue`